_maps/configs/scav.json

```diff
@@ -0,0 +1,27 @@
+{
+	"map_name": "Scav-class Drifter",
+	"map_short_name": "Scav-class",
```

Comment:
"prefix": ISV,
"namelists": insert your namelists of choice here, i'd suggest GENERAL and SPACE at a minimum